### PR TITLE
RUMM-395 dialogs

### PR DESCRIPTION
**Pull request by xgouchet**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/DatadogGesturesTracker.kt [hunk 1]
```diff
@@ -6,22 +6,25 @@
...
 internal class DatadogGesturesTracker(
     internal val targetAttributesProviders: Array<ViewAttributesProvider>
-) :
-    GesturesTracker {
+) : GesturesTracker {
 
     // region GesturesTracker
-    override fun startTracking(activity: Activity) {
+
+    override fun startTracking(window: Window?, context: Context) {
```
Comment (mariusc83) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/DatadogGesturesTracker.kt:21]: now that we changed this I am thinking maybe we should not have an optional there for the Window and instead to put the Nullability condition before calling the function. If this is a big change you can leave it like this and open a ticket instead and I will take care of it next sprint.

File: instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/rum/RumFragmentTrackingPlaygroundActivity.kt [hunk 2]
```diff
@@ -36,7 +36,11 @@ internal class RumFragmentTrackingPlaygroundActivity : AppCompatActivity() {
             .useCustomLogsEndpoint(RuntimeConfig.logsEndpointUrl)
             .useCustomTracesEndpoint(RuntimeConfig.tracesEndpointUrl)
             .useCustomRumEndpoint(RuntimeConfig.rumEndpointUrl)
-            .useViewTrackingStrategy(FragmentViewTrackingStrategy(true))
+            .useViewTrackingStrategy(
```
Comment (mariusc83) [inline instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/rum/RumFragmentTrackingPlaygroundActivity.kt:39]: Editor playing tricks on you ?

Comment (xgouchet) [reply instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/rum/RumFragmentTrackingPlaygroundActivity.kt:39]: Hmm indeed

File: dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/DatadogGesturesTracker.kt (hunk 1 above)
Comment (mariusc83) [reply dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/DatadogGesturesTracker.kt:21]: Never mind this.

File: instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/rum/RumFragmentTrackingPlaygroundActivity.kt (hunk 2 above)
Comment (xgouchet) [reply instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/rum/RumFragmentTrackingPlaygroundActivity.kt]: Fixed

**Pull request merged**